pkg/node/node.go

```diff
@@ -1534,93 +1534,63 @@ func (s *Service) readFCPortsFilterFile() ([]string, error) {
 
 func (s *Service) setupHost(initiators []string, client gopowerstore.Client, arrayIP, arrayID string) error {
 	log.Infof("setting up host on %s", arrayIP)
-	defer log.Infof("finished setting up host on %s", arrayIP)
```

Comment:
isn't it good to have this log in place as it is?

Reply:
I have added the log back